src/client-side-encryption/common.js

```diff
@@ -1,33 +1,4 @@
-/**
```

Comment:
`debug` was moved to the state machine, and the namespace helpers were removed in favor of `MongoDBNamespace`.  the remainder of the file is untouched and I disabled lint because we plan to remove this entirely in a follow-up-ticket.